lib/solidus_dev_support/templates/extension/spec/spec_helper.rb.tt

```diff
@@ -6,7 +6,10 @@ ENV['RAILS_ENV'] = 'test'
 # Run Coverage report
 require 'solidus_dev_support/rspec/coverage'
 
-require File.expand_path('dummy/config/environment.rb', __dir__)
+require File.expand_path('dummy/config/environment.rb', __dir__).tap { |file|
+  # Create the dummy app if it's still missing.
+  system 'bin/rake extension:test_app' unless File.exist? file
```

Comment:
definitely non blocking, but `File.file?` is more specific by looking for a file, `File.exist?` would be true also for a directory

Reply:
Yes, technically I guess dirs are a specialization of files 👍